### DISCUSSION TITLE
Quantitaive performance of DynUNet

### DISCUSSION THREAD

**Discussion by GOZGOXGOYGO** [Project-MONAI/MONAI#1396] [Q&A]
Hi,

may I ask if there is a quantitaive performance comparison between the DynUNet and the nnUNet? Thank you.

**Comment by Nic-Ma** [comment]
Hi @wyli and @ericspod ,

Could you please help share some information here?

Thanks in advance.

**Comment by ericspod** [comment]
I can't say myself anything about the two networks for performance, @wyli or @martaranzini or @irme might have some input.

**Reply by tvercaut** [reply]
Some interesting comparisons can be found in #807 and some further links in #246 
In any case MONAI's dynUNet mostly implements the network part of nnUNet for the moment. Several heuristics relating to hyperparameter choices from the nnU-Net code are yet to be implemented in MONAI.
